docs/usage.md

```diff
@@ -53,6 +53,75 @@ TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,,reverse
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
+## BSJ detection
+
+This part of the pipeline is responsible for the detection of back-splice junctions (BSJs) in the input data. The following tools are currently supported:
+
+- `CIRCexplorer2`
+- `circRNA finder`
+- `CIRIquant`
+- `DCC`
+- `find circ`
+- `MapSplice`
+- `Segemehl`
+
+The tools to be used can be specified using the `tools` parameter.
+Each of the tools also quantifies how many reads support each BSJ. You can specify a cutoff for the minimum number of reads supporting a BSJ using the `bsj_reads` parameter.
+Additionally, the parameter `tool_filter` can be used to specify how many tools a BSJ has to be detected by to be considered as a valid hit.
+
+For instructions on how to interpret the output of this section, please check out the [output documentation](https://nf-co.re/circrna/dev/output#bsj-detection).
+
+## Annotation
+
+The annotation is generally based on the reference GTF file. It can also utilize BED files that are provided by the various circRNA databases.
+The GTF-based annotation allows setting the parameter `exon_boundary` to specify a window around exons. If the BSJ is within this window, it will be annotated as a circRNA - otherwise, it will be annotated as an exon-intron circRNA (EI-circRNA). The default value is 0.
```

Comment:
```suggestion
The GTF-based annotation allows setting the parameter `exon_boundary` to specify a window around exons. If the BSJ is within this window, it will be annotated as a circRNA - otherwise, it will be annotated as an exon-intron circRNA (EI-circRNA). The default value is `0`.
```